tests/diff_test.sh

```diff
@@ -39,6 +39,21 @@ function test_diff_side_by_side()
 
 }
 
+function test_diff_folders()
+{
+  local folder_1="$SAMPLES_DIR/db_files"
+  local folder_2="$SAMPLES_DIR/first_set_of_bytes_from_disk"
+
+  output=$(diff_folders 'an_invalid_file' "$folder_2")
+  assertEquals "($LINENO) Expected 22" 22 "$?"
+
+  output=$(diff_folders 'an_invalid_file' 'another_invalid_file')
+  assertEquals "($LINENO) Expected 22" 22 "$?"
```

Comment:
Split these feature tests into two parts, let's say:
```bash
test_diff_folder_invalid_path()
...
test_diff_folder()
```
Additionally, add a test of two files that have no difference.